### PR TITLE
Add line menu cli

### PR DESCRIPTION
**Pull request by xxhomey19**

### CODE REVIEW

File: src/cli/providers/sh/index.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
+import init from './init';
+import help from './help';
+
 export default {
   title: 'Bottender',
   subcommands: new Set(['help', 'init']),
   get init() {
-    return require('./init').default;
+    return init;
```
Comment (chentsulin) [inline src/cli/providers/sh/index.js]: we should keep those lazy requires because it reduces the start-up time.

**Comment by chentsulin** [comment]
LGTM. @tw0517tw do you want to take a look?

File: src/cli/providers/line/menu.js [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+import pMap from 'p-map';
+import chalk from 'chalk';
+
+import getConfig from '../../shared/getConfig';
+import { print, error, bold, log } from '../../shared/log';
+
+const generateDeleteQuestions = richMenus => [
+  {
+    name: 'deletedRichMenuNames',
+    message: 'Which rich menus do you want to delete ?',
```
Comment (kpman) [inline src/cli/providers/line/menu.js]: maybe no space before `?`

Comment (tw0517tw) [inline src/cli/providers/line/menu.js]: `which rich menu`?

Comment (tw0517tw) [reply src/cli/providers/line/menu.js]: Ah, it's checkbox...

File: src/cli/providers/line/menu.js [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+
+export const help = () => {
+  console.log(`
+    bottender line menu <action> [option]
+
+    ${chalk.dim('Actions:')}
+
+      get           Get the LINE rich menu
+      set           Set the LINE rich menu by diff
+      del, delete   Delete all the LINE rich menu
```
Comment (tw0517tw) [inline src/cli/providers/line/menu.js]: will
```
Get LINE rich menus
Set LINE rich menu by diff
Delete all LINE rich menus
```
be better?

Comment (tw0517tw) [reply src/cli/providers/line/menu.js]: also remove `the` in `the LINE rich menu` below

File: src/cli/providers/line/menu.js [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+    const config = getConfig('bottender.config.js', 'line');
+
+    invariant(config.accessToken, 'accessToken is not found in config file.');
+
+    const client = LineClient.connect(config.accessToken);
+
+    const richMenus = await client.getRichMenuList();
+
+    if (richMenus) {
+      print('The rich menu is:');
```
Comment (tw0517tw) [inline src/cli/providers/line/menu.js]: `the rich menus are`?

File: src/cli/providers/telegram/__tests__/index.spec.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import sh from '..';
+
+jest.mock('../webhook');
+jest.mock('../help');
+
+describe('LINE cli', () => {
```
Comment (kpman) [inline src/cli/providers/telegram/__tests__/index.spec.js]: telegram

File: src/cli/providers/line/menu.js [hunk 6]
```diff
@@ -0,0 +1,260 @@
...
+      error(err.message);
+    }
+    return process.exit(1);
+  }
+}
+
+export async function setLineMenus(ctx) {
+  const { force } = ctx.argv;
+  try {
+    const { accessToken, richMenus: _richMenus } = getConfig(
```
Comment (tw0517tw) [inline src/cli/providers/line/menu.js]: how about name this richMenus as `localRichMenus` and the ones from LINE API `onlineRichMenus`

File: src/cli/providers/sh/__tests__/index.spec.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
+import sh from '..';
+
+jest.mock('../init');
+jest.mock('../help');
+
+describe('LINE cli', () => {
```
Comment (tw0517tw) [inline src/cli/providers/sh/__tests__/index.spec.js]: LINE?

File: src/cli/providers/telegram/__tests__/index.spec.js (hunk 5 above)
Comment (tw0517tw) [inline src/cli/providers/telegram/__tests__/index.spec.js]: LINE?

File: src/cli/providers/line/menu.js (hunk 6 above)
Comment (kpman) [reply src/cli/providers/line/menu.js]: I think it is good to name this richMenus as `localRichMenus`. agree with @tw0517tw

File: src/cli/providers/line/menu.js [hunk 8]
```diff
@@ -26,12 +26,12 @@ export const help = () => {
...
       get           Get the LINE rich menus
       set           Set the LINE rich menu by diff
-      del, delete   Delete all the LINE rich menus
+      del, delete   Delete the LINE rich menus
       help          Show this help
 
     ${chalk.dim('Options:')}
 
-      --force       Force update the LINE rich menu by config
+      --force       With action del, force delete ALL LINE rich menu
```
Comment (kpman) [inline src/cli/providers/line/menu.js]: menus or not?

**Pull request merged**